manifests/client/add_cmd_job.pp

```diff
@@ -0,0 +1,45 @@
+# @summary Defined type to add a new service backup cmd job
+#
+# Add a service cmd backup job to this backup client
+# This allows the streaming of data (STDOUT) directly to
+# the backup without staging data to a file first. An example
+# for the use would be to eliminate the need for a large database
+# to be stored locally before being backed up.
+#
+# @param backup_command
+#   List of directory paths for the job to backup.
+#   Can be a list of directories and/or specific files.
+#
+# @param filename
+#   Name of the file created by the command located 
+#   within the archive.
+#
+# @param prehook_commands
+#   Optional list of commands to run before backup job
+#
+# @param posthook_commands
+#   Optional list of commands to run after the backup job
+#
+# @example
```

Comment:
This is where I’m looking for an example of how to use this type.

Reply:
I've updated it.

Reply:
I've updated the example.